frontend/main.css

```diff
@@ -0,0 +1,22 @@
+#map {
+	height: 70vh;
```

Comment:
not to start another [flame war here](https://thenewstack.io/spaces-vs-tabs-a-20-year-debate-and-now-this-what-the-hell-is-wrong-with-go/) 😂

but might be worth noting GDS way prefers two spaces over tabs for both [CSS](https://gds-way.cloudapps.digital/manuals/programming-languages/css.html#whitespace) and[ javascript](https://gds-way.cloudapps.digital/manuals/programming-languages/js.html#whitespace)

it's a pedantic thing but means that if you get really deeply nested stuff or longer lines it's much easier to read.

here's a guide on how to quickly fix that in VS code

https://bobbyhadz.com/blog/vscode-change-indentation

Reply:
Fixed!